### PR TITLE
[luci/import] Introduce copy_data_4

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce common copy_data_4 for S4/U4.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by jinevening** [comment]
Is this for reducing duplicate code?

**Comment by seanshpark** [comment]
> Is this for reducing duplicate code?

Yup :)

File: compiler/luci/import/src/Nodes/CircleConst.cpp [hunk 1]
```diff
@@ -150,6 +102,33 @@ void copy_data<loco::DataType::STRING>(const VectorWrapper<uint8_t> &raw_data,
...
+
+  const uint8_t *data = raw_data.data();
+  const_node->size<DT>(num_elements);
+  for (uint32_t i = 0; i < raw_size; ++i)
+  {
+    uint32_t idx = i * 2;
+    // 1bit for sign, 3bit for value
+    const_node->at<DT>(idx++) = static_cast<T>(data[i] << 4) >> 4;
+    if (idx < num_elements)
+      const_node->at<DT>(idx) = static_cast<T>(data[i]) >> 4;
```
Comment (mhs4670go) [inline compiler/luci/import/src/Nodes/CircleConst.cpp:128]: Maybe?
```suggestion
      const_node->at<DT>(idx) |= static_cast<T>(data[i]) >> 4;
```

File: compiler/luci/import/src/Nodes/CircleConst.cpp [hunk 2]
```diff
@@ -150,6 +102,33 @@ void copy_data<loco::DataType::STRING>(const VectorWrapper<uint8_t> &raw_data,
...
+
+  uint32_t raw_size = (num_elements + 1) / 2;
+  assert(raw_data.size() == raw_size);
+
+  const uint8_t *data = raw_data.data();
+  const_node->size<DT>(num_elements);
+  for (uint32_t i = 0; i < raw_size; ++i)
+  {
+    uint32_t idx = i * 2;
+    // 1bit for sign, 3bit for value
```
Comment (mhs4670go) [inline compiler/luci/import/src/Nodes/CircleConst.cpp:125]: Can this comment apply to U4 as well? I guess 4 bit for value when it comes to the U4.

**Review by mhs4670go — APPROVED** [review]

File: compiler/luci/import/src/Nodes/CircleConst.cpp (hunk 2 above)
Comment (jinevening) [inline compiler/luci/import/src/Nodes/CircleConst.cpp:125]: I think this comment has to be removed. U4 does not have a sign bit.

**Comment by mhs4670go** [comment]
Ah, I mistakenly approved the PR.

**Review by jinevening — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**